### ISSUE TITLE
azurerm_cosmosdb_sql_container indexing_policy marks changes when it should not 

### ISSUE THREAD

**Issue by katlimruiz** [terraform-providers/terraform-provider-azurerm#9294]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version
Terraform v0.13.5

### Affected Resource(s)
* `azurerm_cosmosdb_sql_container`

### Terraform Configuration Files
```hcl
resource "azurerm_cosmosdb_sql_container" "cdbsql_myobject" {
  name                = "MyObject"
  resource_group_name = xx
  account_name        = xx
  database_name       = xx
  partition_key_path  = xx

  indexing_policy {
    indexing_mode = "Consistent"

    included_path {
      path = "/\"objectName\"/?"
    }

    included_path {
      path = "/\"objectUrl\"/?"
    }

    excluded_path {
      path = "/*"
    }
    excluded_path {
      path = "/\"_etag\"/?"
    }
  }
}
```

### Expected Behaviour

No changes should be detected when the file has not changed

### Actual Behaviour

TF detects changes on the indexing_policy definition, and the output is the following:

```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # module.production.module.cdbsql.azurerm_cosmosdb_sql_container.cdbsql_catalogdb_brand will be updated in-place
  ~ resource "azurerm_cosmosdb_sql_container" "cdbsql_myobject" {
          ...
      ~ indexing_policy {
            indexing_mode = "consistent"

            excluded_path {
                path = "/*"
            }
          + excluded_path {
              + path = "/\"_etag\"/?"
            }

            included_path {
                path = "/\"objectName\"/?"
            }
            included_path {
                path = "/\"objectUrl\"/?"
            }
        }
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

### Steps to Reproduce
1. I already have my objects in Azure. 
2. WITHOUT making any change to the files, I run `terraform plan`, and it **detects changes**.
3. I also run `terraform plan -target=module.production.module.cdbsql.azurerm_cosmosdb_sql_container.cdbsql_myobject` and it also "detects" the change when it should not.

**Comment by katlimruiz** [comment]
and this is specially painful since I have around 30+ objects in cosmosdb so it takes more time to run `plan` and `apply`

**Comment by katlimruiz** [comment]
Ok, so I got it fixed by removing 
```
    excluded_path {
      path = "/\"_etag\"/?"
    }
```

from the definition. 

(this is my explanation) So it seems driver? provider? resource? already adds this automatically so if I do it explictly, it kind of takes it as something I am additionally doing, and therefore overriding the defautl.

To me, this is very counterintuitive, since if I set the same as the default, then it should be ok and not take it as a change. For me at least this is still considered as a bug.

HOWEVER, I had the same issue with mssql database and license_type field. I removed it from my definition (since it also had the same default), and it stopped marking it as a change.

Well hope this helps someone else.
